package.json

```diff
@@ -19,7 +19,8 @@
     "db:reset": "prisma migrate reset",
     "db:format": "prisma format",
     "db:studio": "prisma studio",
-    "setup": "run-s db:generate db:push db:seed",
+    "db:setup": "run-s db:generate db:push",
+    "setup": "run-s db:setup db:seed",
```

Comment:
Nit. I'm wondering if there are better names to distinguish the two setups. They are both database related, and the only difference is whether we seed, not whether it's database related (`db`).
Maybe `db:setup` can be `setup:seedless`?

Reply:
I'm not actually using this in my code, so I think I'll revert it back to what it was.